test/include/rev-printf.h

```diff
@@ -1,17 +1,42 @@
-// See LICENSE for license details.
-
+//
+// _rev_printf_h_
+//
+// Copyright (C) 2017-2025 Tactical Computing Laboratories, LLC
+// All Rights Reserved
+// contact@tactcomplabs.com
+//
+// See LICENSE in the top level directory for licensing details
+//
+
+// Notes
+// - See $REVHOME/test/syscall/printf for example usage
+// - If the version of printf in stdio is called then rev
+//   should print an error message for an unimplemented ecall.
+
+#ifndef __REV_PRINTF_H__
+
+// #define REV_DEBUG
+#ifdef REV_DEBUG
+#define dprintf rev_fast_printf
+#else
+#define dprintf
```

Comment:
A different name like `debug_printf()` should be used than `dprintf()`, since `dprintf()` is a POSIX function like `fprintf()` but to file descriptors like `STDOUT_FILENO=1` and `STDERR_FILENO=2`.